test/adaptive_load/fake_plugins/fake_input_variable_setter/fake_input_variable_setter.cc

```diff
@@ -0,0 +1,70 @@
+#include "test/adaptive_load/fake_plugins/fake_input_variable_setter/fake_input_variable_setter.h"
+
+namespace Nighthawk {
+
+namespace {
+
+absl::Status StatusFromProtoRpcStatus(const google::rpc::Status& status_proto) {
+  return absl::Status(static_cast<absl::StatusCode>(status_proto.code()), status_proto.message());
+}
+
+} // namespace
+
+FakeInputVariableSetter::FakeInputVariableSetter(
+    const nighthawk::adaptive_load::FakeInputVariableSetterConfig& config)
+    : adjustment_factor_{config.adjustment_factor() > 0 ? config.adjustment_factor() : 1} {}
+
+absl::Status FakeInputVariableSetter::SetInputVariable(
```

Comment:
const double?

Reply:
Since this is passed by value we probably don't need const.